src/fprime/fbuild/settings.py

```diff
@@ -54,7 +54,7 @@ def read_safe_path(
         all_paths = parser.get(section, key, fallback="").split(":")
         expanded = []
         for path in all_paths:
-            if path == "" or path is None:
```

Comment:
`if not path` is sufficient here....as a None is also falsey.

Reply:
Correct!